pkg/health/membergarbagecollector/membergarbagecollector_suite_test.go

```diff
@@ -0,0 +1,37 @@
+package membergarbagecollector_test
+
+import (
+	"os"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	logger = logrus.New().WithField("suite", "snapshotter")
```

Comment:
Do we need log messages in tests? Otherwise we can use `logger.Out = io.Discard` to not pollute the test output.

Reply:
Done